torch/_dynamo/test_case.py

```diff
@@ -29,13 +30,24 @@
 log = logging.getLogger(__name__)
 
 
+def check_skip_windows_inductor_UTs_switch() -> bool:
+    if not IS_WINDOWS:
+        return False
+
+    var = os.environ.get("RUN_ALL_WIN_INDUCTOR_UTS", "OFF")
+    if var in ["ON", "YES", "1", "Y"]:
+        return False
+
+    return True
+
+
 def run_tests(needs: Union[str, tuple[str, ...]] = ()) -> None:
     from torch.testing._internal.common_utils import run_tests
 
     if TEST_WITH_TORCHDYNAMO or TEST_WITH_CROSSREF:
         return  # skip testing
 
-    if not torch.xpu.is_available() and IS_WINDOWS:
+    if not torch.xpu.is_available() and check_skip_windows_inductor_UTs_switch():
```

Comment:
```suggestion
    if not torch.xpu.is_available() and IS_WINDOWS and os.environ.get("TORCHINDUCTOR_WINDOWS_TESTS", "0") == "0":
```

I don't think we need the helper above.